Dockerfile

```diff
@@ -35,10 +35,13 @@ RUN apt-get update && apt-get install -y \
     gcc \
```

Comment:
might be good to set the `--no-install-recommends` flag, and be specific about the exact dependencies to install (which _could_ explain the difference

Reply:
Added the flag, thanks.